src/styles/blocks/card.scss

```
@@ -0,0 +1,90 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  &_img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &_description {
+    color: #{$color-text-secondary};
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &_code {
+    color: #{$color-text-primary};
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &_review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    width: 166px;
```

No need to hardcode width for review block
```suggestion
    width: 100%;
```